dsx-fw/DSX_Library/DAC/Src/DAC.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * DAC.c
+ *
+ *  Created on: May 28, 2021
+ *      Author: Manraj Sangha
+ */
+
+#include "DAC.h"
+
+//Function to initialize DAC Channel
+void DAC_init(DAC_HandleTypeDef* hdac, uint32_t CHANNEL){
+	HAL_DAC_Start(hdac, CHANNEL);
+}
+
+//Function to write to DAC Channel
+//Input value must be 12-bit right aligned
+void DAC_write(DAC_HandleTypeDef* hdac, uint32_t val, uint32_t CHANNEL){
```

Comment:
Can you make the DAC_write not input the adc handle and instead the adc handle accessed in your local file scope? It's one less thing to include in the commands library. The DAC_init can stay the same.

suggestion:
```C
void DAC_write(uint32_t val, uint32_t CHANNEL)
```